services/src/main/java/com/linkedin/kmf/services/configs/ConsumeServiceConfig.java

```diff
@@ -42,6 +42,10 @@
   public static final String CONSUMER_PROPS_CONFIG = "consume.consumer.props";
   public static final String CONSUMER_PROPS_DOC = "The properties used to config consumer in consume service.";
 
+  public static final String LATENCY_SLA_MS_CONFIG = "consume.latency.sla.ms";
```

Comment:
Do you think `consume.latency.sla.ms` is a good config name? Currently the variable name is named after the `consume.` part, e.g. LATENCY_SLA_MS for `.latency.sla.ms`. Because all we already know that variables in `ConsumeServiceConfig.java` are used for ConsumeService.